lib/loader.js

```diff
@@ -101,4 +90,36 @@ function sassLoader(content) {
     });
 }
 
+/**
+ * Verifies that the implementation and version of Sass is supported by this loader.
+ *
+ * @param {string} info
+ */
+function validateSassVersion(info) {
+    const parsed = /^([^\t]+)\t([^\t]+)/.exec(info);
```

Comment:
Don't use regex here, please use `semver` for parsing `major` and `minor`

Reply:
Done.

Reply:
@nex3 looks we still use `/^([^\t]+)\t([^\t]+)/.exec(info)`, better use `semver.major`, `semver.minor` and `semver.patch` functions

Reply:
@nex3 maybe we can do this without regex?